integreat_cms/cms/templates/pois/poi_form.html

```diff
@@ -33,7 +33,11 @@ <h1 class="heading overflow-hidden text-ellipsis">
             {% if not poi_form.instance.archived and perms.cms.change_poi %}
                 <div class="flex flex-wrap gap-4 ml-auto mr-0">
                     <button name="status" value="{{ DRAFT }}" class="btn btn-gray">{% translate "Save as draft" %}</button>
-                    <button name="status" value="{{ PUBLIC }}" class="btn">
+                    <button name="status"
+                            value="{{ PUBLIC }}"
+                            class="btn"
+                            {% if poi_form.instance.id %} {% get_public_translation poi_form.instance request.region.default_language.slug as poi_translation %} {% if language != request.region.default_language and not poi_translation %} title="{% translate "The default translation is not yet published" %}" disabled {% endif %}
+                            {% endif %}>
```

Comment:
```suggestion
                            {% if language != request.region.default_language and not poi_form.instance.default_public_translation %} title="{% translate "The default translation is not yet published" %}" disabled {% endif %}>
```
Another option would be to use the helper property `default_public_translation` to simplify the condition.